### PR TITLE
Add unstable property to quickstart docs

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Solves #841

- patch: Add debug: unstable to docs
- Make always-fail error more descriptive

### CODE REVIEW

File: suites/e2e/tests/always-fail/index.js [hunk 1]
```diff
@@ -38,7 +40,7 @@ module.exports = {
         await this.worker.instantKill("💥");
         test.notOk(
           true,
-          `This test should have been skipped in the first place but didn't`,
+          `This test should have been skipped in the first place but didn't. Add this test to the unstable list avaiable in the debug object in config.js. Refer to debugging docs or test code for more information.`,
```
Comment (lmbarros) [inline suites/e2e/tests/always-fail/index.js:43]:
```suggestion
          `This test should have been skipped in the first place but didn't. Add this test to the unstable list available in the debug object in config.js. Refer to debugging docs or test code for more information.`,
```

**Review by lmbarros — CHANGES_REQUESTED** [review]
Thanks, Vipul! I spotted a small typo, but otherwise it's good to merge IMO!

**Comment by vipulgupta2048** [comment]
@lmbarros resolved, please re-review

**Review by lmbarros — APPROVED** [review]
:rocket:

**Comment by vipulgupta2048** [comment]
@balena-ci rebase

**Comment by vipulgupta2048** [comment]
@balena-ci I self-certify!

**Comment by vipulgupta2048** [comment]
The bulldozer maybe undergoing an outage so merging it myself.

**Pull request merged**